constants/constants.go

```diff
@@ -0,0 +1,16 @@
+package constants
+
+const (
+	TaxRateForRAW                                    = 12.50
+	ImportDuty                                       = 10.00
+	ImportDutyLimit1                                 = 100
+	ImportDutyLimit2                                 = 200
+	SurchargeAmountForFinalCostUptoImportDutyLimit1  = 5
+	SurchargeAmountForFinalCostUptoImportDutyLimit2  = 10
+	SurchargeRateForFinalCostExceedeImportDutyLimit2 = 5.00
```

Comment:
naming can be improved.

Reply:
improved the naming (ImportDutyLimit2SurchargeAmt)